src/components/AdvancedFacetViewModal/style.css

```diff
@@ -1,32 +1,10 @@
-.advancedFacetsOverlay {
-  position: absolute;
-  left: 0px;
-  right: 0px;
-  top: 0px;
-  bottom: 0px;
-  background: rgba(0, 0, 0, 0.5);
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  z-index: 1000;
-}
-.advacnedFacetsContainer {
-  width: 1000px;
-  height: 800px;
-  position: relative;
-  background: white;
-  border-radius: 5px;
-  display: flex;
-  flex-direction: column;
-  overflow: hidden;
-}
-.advacnedFacetsContainer .advancedFacetsWrapper {
+.advancedFacetsWrapper {
```

Comment:
can we try to use `emotion` css instead of a mix of real css and `emotion`? 😅Unless these classes are set in arranger?